<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_animationgroup</class>
 <widget class="QWidget" name="toolbar_animationgroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="cleanListButton">
     <property name="text">
      <string>Clean animation list</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="scansAnimPushButton">
     <property name="text">
      <string>Scans list</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="formatComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="startAnimationButton">
     <property name="text">
      <string>Start animation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="animationSpeedSlider">
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="stopAnimationButton">
     <property name="text">
      <string>Stop animation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="recordPerfCheckBox">
     <property name="text">
      <string>Record Perf</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="loopCheckBox">
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
